test-requirements.txt.in

```diff
@@ -9,3 +9,4 @@ more-itertools==5.0.0 #using older version to preserve Python 2.7 support.
 mock==2.0.0
 pytest==3.7.1
 futures==3.1.1
+psycopg2==2.8.3
```

Comment:
The `.in` file generally shouldn't pin exact versions. It should just list package names and let `update-requirements` decide what versions to put in the `.txt` files. The only time versions should appear here is for things like more-itertools or sqlalchemy where freely upgrading versions would be problematic.